gm4_moneo_shamir/data/gm4_moneo_shamir/predicates/holding_moneo.json

```diff
@@ -1,29 +1,12 @@
 {
-  "condition": "minecraft:alternative",
-  "terms": [
-    {
-      "condition": "minecraft:entity_properties",
-      "entity": "this",
-      "predicate": {
-        "equipment": {
-          "mainhand": {
-            "tag": "gm4_moneo_shamir:tools",
-            "nbt": "{gm4_metallurgy:{has_shamir:1b,active_shamir:'moneo'}}"
-          }
-        }
-      }
-    },
-    {
-      "condition": "minecraft:entity_properties",
-      "entity": "this",
-      "predicate": {
-        "equipment": {
-          "offhand": {
-            "tag": "gm4_moneo_shamir:tools",
-            "nbt": "{gm4_metallurgy:{has_shamir:1b,active_shamir:'moneo'}}"
```

Comment:
Players can also use up durability of tools in the offhand, for example by stripping wood with an axe.

Reply:
I removed the offhand check as the rest of the code did not use it at all; `active_tool` only checks for the mainhand, so the offhand was never being checked. Additionally, mining fatigue would not have an effect on right click abilities, so it doesn't really matter to check the offhand for that.